issue.go

```diff
@@ -867,6 +867,33 @@ func (s *IssueService) AddWorklogRecord(issueID string, record *WorklogRecord, o
 	return responseRecord, resp, nil
 }
 
+// AddWorklogRecord adds a new worklog record to issueID.
```

Comment:
This comment doesn't match the method name

Reply:
Oh, I changed it.